index.bs

```diff
@@ -204,7 +204,7 @@ An attribution source is a [=struct=] with the following items:
 : <dfn>aggregatable budget consumed</dfn>
 :: A non-negative integer, total [=aggregatable contribution/value=] of all [=aggregatable contributions=] created with this [=attribution source=].
 : <dfn>aggregatable dedup keys</dfn>
-:: [=ordered set=] of [=attribution trigger/aggregatable dedup keys=] associated with this [=attribution source=].
+:: [=ordered set=] of [=aggregatable dedup key/dedup key|aggregatable dedup keys=] associated with this [=attribution source=].
```

Comment:
This makes it seem like the source needs to retain the filter information for the deduplication keys, but I don't think it does.

Reply:
Updated to `aggregatable dedup key values`, hopefully it's a bit clearer.